grails-app/services/au/org/emii/portal/WpsService.groovy

```diff
@@ -27,4 +37,36 @@ class WpsService extends AsyncDownloadService {
     def onResponseSuccess = { resp, xmlReader ->
         return new groovy.xml.StreamingMarkupBuilder().bindNode(xmlReader).toString()
     }
+
+    def notifyViaEmail(params) {
```

Comment:
Scratch that comment from here, look at `jobComplete` in the controller, but same comment.